support/v1.19.0-sprint-33/update-proclamation-secondary-stages.sql

```
@@ -0,0 +1,114 @@
+                SELECT 1 FROM dristi_task t WHERE t.filingNumber = dc.filingNumber AND t.taskType = 'WARRANT'
+                  AND t.status NOT IN ('EXECUTED','EXPIRED','DELIVERED','ABATED','UNDELIVERED','NOT_EXECUTED')
+            )
+        )
+        THEN true ELSE false END AS has_warrant,
+
+        -- Proclamation & Attachment: published ATTACHMENT order AND not all tasks completed
+        CASE WHEN EXISTS (
+            SELECT 1 FROM dristi_orders o
+            WHERE o.filingNumber = dc.filingNumber AND o.status ILIKE 'published'
+              AND (o.orderType = 'ATTACHMENT'
+                   OR (o.orderCategory = 'COMPOSITE' AND EXISTS (SELECT 1 FROM jsonb_array_elements(o.compositeItems) elem WHERE elem->>'orderType' IN ('ATTACHMENT'))))
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)